release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -499,12 +499,25 @@ For more information, see xref:../machine_management/control_plane_machine_manag
 With this release, {product-title} routes can be configured with third-party certificate management solutions, utilising the `.spec.tls.externalCertificate` field in the route API. This allows you to reference externally managed TLS certificates through secrets, streamlining the process by eliminating manual certificate management. By using externally managed certificates, you reduce errors, ensure a smoother certificate update process, and enable the OpenShift router to promptly serve renewed certificates. For more information, see xref:../networking/routes/secured-routes.adoc#nw-ingress-route-secret-load-external-cert_secured-routes[Creating a route with externally managed certificate].
 
 [id="ocp-4-19-networking-gateway-api-controller_{context}"]
-==== Support for using the Gateway API to configure cluster Ingress traffic
-We will have details here when {product-title} {product-version} is released.
+==== Support for using the Gateway API to configure cluster Ingress traffic (General Availability)
+With this release, support for managing Ingress cluster traffic using Gateway API resources is Generally Available. Gateway API provides a robust networking solution within the transport layer, L4, and the application layer, L7, for {product-title} clusters using a standardized open source ecosystem.
+
+For more information, see xref:../networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc#ingress-gateway-api[Gateway API with {product-title} networking].
+
+[IMPORTANT]
+====
+Gateway API resources must conform to the supported {product-title} API surface. This means you cannot use another vendor-specific resource, such as Istio's VirtualService, with {product-title}'s implementation of the Gateway API. For more information, see xref:../networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc#nw-ingress-gateway-api-implementation[Gateway API implementation for {product-title}].
+====
 
 [id="ocp-4-19-networking-gateway-api-crd-lifecycle_{context}"]
 ==== Support for managing the Gateway API custom resource definition (CRD) lifecycle
-We will have details here when {product-title} {product-version} is released.
+With this release, {product-title} manages the lifecycle of Gateway API CRDs. This means that the Ingress Operator handles the required versioning and management of resources. Any Gateway API resources created in a previous {product-title} version must be recreated and redeployed so that it conforms to the specifications required by the Ingress Operator.
+
+For more information, see xref:../updating/preparing_for_updates/updating-cluster-prepare.adoc#nw-ingress-gateway-api-manage-succession[Preparing for Gateway API management succession by the Ingress Operator].
```

Comment:
Is this link right? It goes to the top of Preparing to update to OpenShift Container Platform 4.19. Maybe the section is forthcoming?